js/test/Ice/properties/Client.ts

```diff
@@ -97,6 +97,17 @@ export class Client extends TestHelper {
             }
             out.writeLine("ok");
         }
+
+        {
+            out.write("testing property regexp pattern...");
+            const properties = Ice.createProperties();
+            properties.setProperty("Ice.Default.Locator.Context.Foo", "Bar");
+            const ctx = properties.getPropertiesForPrefix("Ice.Default.Locator.Context");
+            test(ctx.size == 1);
+            const value = properties.getProperty("Ice.Default.Locator.Context.Foo");
```

Comment:
Not the best tests, but we would at least see a warning if the regexp pattern doesn't match the property.